File: tails.py
import sys

name = sys.argv[1]
file_name = sys.argv[2]
lines = sys.argv[3]

# open file, readlines, and assign list to readMe
readMe = open(file_name).readlines()
#print(readMe)

# uses the slice notation with the negative integer variable (lines) to get the last elements in the list
lines = int(lines)
last_line_slice = readMe[-(lines):]
print(last_line_slice)

File: tails.rb
name = ARGV[0]
file_name = ARGV[1]
lines = ARGV[2]
lines = lines.to_i

#puts name
#puts file_name
#puts lines

# reads file_name and display last lines of the file
readMe = IO.readlines(file_name).last(lines)
puts readMe